src/os/debug/fancierwindow.js

```diff
@@ -5,9 +5,10 @@ goog.require('goog.debug.FancyWindow');
 goog.require('goog.html.SafeStyleSheet');
 goog.require('goog.string.Const');
 goog.require('os.alert.AlertManager');
-goog.require('os.file.persist.FilePersistence');
 goog.require('os.ui');
```

Comment:
This file still has a bunch of naked requires in it. Doesn't look like it was explicitly converted as part of this ticket, but we'll need to hit it eventually

Reply:
Good call, this probably got transformed before I created the `replaceglobals` script. It's updated now.